msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2018-04-02 17:36+AKDT\n"
"PO-Revision-Date: 2018-04-15 16:43-0400\n"
"Last-Translator: Kowlin <boxedpp@gmail.com>\n"
"Language-Team: Chinese Simplified\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Language: zh-CN\n"
"X-Crowdin-File: /core/locales/messages.pot\n"
"Language: zh_CN\n"

#: ../cog_manager.py:313
msgid "Install Path: {}\n\n"
msgstr "安装路径：{}\n\n"

#: ../cog_manager.py:329
msgid "That path does not exist or does not point to a valid directory."
msgstr ""

#: ../cog_manager.py:338
msgid "Path successfully added."
msgstr "路径已成功添加。"

#: ../cog_manager.py:351
msgid "That is an invalid path number."
msgstr "这是一个无效的路径编号。"

#: ../cog_manager.py:355
msgid "Path successfully removed."
msgstr "路径已被成功删除。"

#: ../cog_manager.py:371
msgid "Invalid 'from' index."
msgstr "无效的‘从’索引"

#: ../cog_manager.py:377
msgid "Invalid 'to' index."
msgstr "无效的‘到’索引"

#: ../cog_manager.py:381
msgid "Paths reordered."
msgstr "已重新排序路径。"

#: ../cog_manager.py:399
msgid "That path does not exist."
msgstr "该路径不存在。"

#: ../cog_manager.py:403
msgid "The bot will install new cogs to the `{}` directory."
msgstr "机器人将安装新的齿轮到`{}`目录。"

#: ../core_commands.py:171
msgid "Embeds are now {} by default."
msgstr ""

#: ../core_commands.py:193 ../core_commands.py:218
msgid "Embeds will now fall back to the global setting."
msgstr ""

#: ../core_commands.py:197
msgid "Embeds are now {} for this guild."
msgstr ""

#: ../core_commands.py:222
msgid "Embeds are now {} for you."
msgstr ""

#: ../core_commands.py:483
msgid "Shutting down... "
msgstr "关闭... "

#: ../core_commands.py:498
msgid "Restarting..."
msgstr ""

#: ../core_commands.py:556
msgid "The admin role for this guild has been set."
msgstr "针对此行会的管理员角色已经设置。"

#: ../core_commands.py:564
msgid "The mod role for this guild has been set."
msgstr "针对此行会的协调员角色已经设置。"

#: ../core_commands.py:577
msgid "Failed. Remember that you can edit my avatar up to two times a hour. The URL must be a direct link to a JPG / PNG."
msgstr "失败。请记住在每个小时你最多可以编辑两次头像。网址必须是JPG/PNG图像。"

#: ../core_commands.py:581
msgid "JPG / PNG format only."
msgstr "支持 JPG/PNG 格式。"

#: ../core_commands.py:583 ../core_commands.py:680 ../core_commands.py:694
msgid "Done."
msgstr "完成。"

#: ../core_commands.py:598
msgid "Game set."
msgstr "游戏集。"

#: ../core_commands.py:613
msgid "Listening set."
msgstr ""

#: ../core_commands.py:628
msgid "Watching set."
msgstr ""

#: ../core_commands.py:657
msgid "Status changed to {}."
msgstr ""

#: ../core_commands.py:689
msgid "Failed to change name. Remember that you can only do it up to 2 times an hour. Use nicknames if you need frequent changes. `{}set nickname`"
msgstr "无法更改名字。请记住每个小时你只能最多进行两次操作。如果需要频繁更改，请使用昵称。`{}设置昵称`"

#: ../core_commands.py:704
msgid "I lack the permissions to change my own nickname."
msgstr "我没有更改我自己昵称的权限。"

#: ../core_commands.py:718 ../core_commands.py:731
msgid "Prefix set."
msgstr "前缀集。"

#: ../core_commands.py:727
msgid "Guild prefixes have been reset."
msgstr "已重置行会前缀。"

#: ../core_commands.py:750
msgid "\n"
"Verification token:"
msgstr "验证令牌："

#: ../core_commands.py:753
msgid "Remember:\n"
msgstr "记住："

#: ../core_commands.py:756
msgid "I have printed a one-time token in the console. Copy and paste it here to confirm you are the owner."
msgstr "我在控制台打印了一次性的令牌。复制并粘贴到这里以确认你是所有者。"

#: ../core_commands.py:764
msgid "The set owner request has timed out."
msgstr "设置所有者请求已经超时。"

#: ../core_commands.py:770
msgid "You have been set as owner."
msgstr "你已被设置为所有者。"

#: ../core_commands.py:772
msgid "Invalid token."
msgstr "无效令牌。"

#: ../core_commands.py:787
msgid "Please use that command in DM. Since users probably saw your token, it is recommended to reset it right now. Go to the following link and select `Reveal Token` and `Generate a new token?`.\n\n"
"https://discordapp.com/developers/applications/me/{}"
msgstr ""

#: ../core_commands.py:810
msgid "Locale has been set."
msgstr "区域已经被设置。"

#: ../core_commands.py:824
msgid "Done. Sentry logging is now enabled."
msgstr ""

#: ../core_commands.py:827
msgid "Done. Sentry logging is now disabled."
msgstr ""

#: ../core_commands.py:880
msgid "A backup has been made of this instance. It is at {}."
msgstr ""

#: ../core_commands.py:884
msgid "That directory doesn't seem to exist..."
msgstr ""

#: ../core_commands.py:894
msgid "User ID: {}"
msgstr ""

#: ../core_commands.py:897
msgid "through DM"
msgstr "通过DM"

#: ../core_commands.py:899
msgid "from {}"
msgstr "从 {}"

#: ../core_commands.py:900
msgid " | Server ID: {}"
msgstr ""

#: ../core_commands.py:909
msgid "Use `{}dm {} <text>` to reply to this user"
msgstr "使用 \"{} dm {} <text>\" 来答复此用户"

#: ../core_commands.py:912
msgid "Sent by {} {}"
msgstr "发送人 {} {}"

#: ../core_commands.py:930 ../core_commands.py:943
msgid "I cannot send your message, I'm unable to find my owner... *sigh*"
msgstr "我不能发送你的信息。我无法找到我的所有人... *sigh*"

#: ../core_commands.py:933 ../core_commands.py:946
msgid "I'm unable to deliver your message. Sorry."
msgstr "抱歉，我无法发送你的消息。"

#: ../core_commands.py:935 ../core_commands.py:948
msgid "Your message has been sent."
msgstr "你的信息已发送。"

#: ../core_commands.py:962
msgid "Invalid ID or user not found. You can only send messages to people I share a server with."
msgstr "非法ID或者找不到用户。你只能向和我共享服务器的人发送消息。"

#: ../core_commands.py:970
msgid "Owner of {}"
msgstr ""

#: ../core_commands.py:971
msgid "You can reply to this message with {}contact"
msgstr ""

#: ../core_commands.py:984 ../core_commands.py:993
msgid "Sorry, I couldn't deliver your message to {}"
msgstr ""

#: ../core_commands.py:987 ../core_commands.py:996
msgid "Message delivered to {}"
msgstr ""

#: ../core_commands.py:1016
msgid "User added to whitelist."
msgstr ""

#: ../core_commands.py:1025
msgid "Whitelisted Users:"
msgstr ""

#: ../core_commands.py:1045
msgid "User has been removed from whitelist."
msgstr ""

#: ../core_commands.py:1047
msgid "User was not in the whitelist."
msgstr ""

#: ../core_commands.py:1055
msgid "Whitelist has been cleared."
msgstr ""

#: ../core_commands.py:1072
msgid "You cannot blacklist an owner!"
msgstr ""

#: ../core_commands.py:1079
msgid "User added to blacklist."
msgstr ""

#: ../core_commands.py:1088
msgid "blacklisted Users:"
msgstr ""

#: ../core_commands.py:1108
msgid "User has been removed from blacklist."
msgstr ""

#: ../core_commands.py:1110
msgid "User was not in the blacklist."
msgstr ""

#: ../core_commands.py:1118
msgid "blacklist has been cleared."
msgstr ""

#: ../dev_commands.py:213
msgid "Already running a REPL session in this channel. Exit it with `quit`."
msgstr "已经在这个通道中运行REPL会话。选择'退出'来退出。"

#: ../dev_commands.py:218
msgid "Enter code to execute or evaluate. `exit()` or `quit` to exit."
msgstr "输入代码来执行或评估。`exit()` 或 `quit`来退出。"

#: ../dev_commands.py:281
msgid "Unexpected error: `{}`"
msgstr "异常错误：`{}`"

